inlong-dashboard/src/core/stores/index.ts

```diff
@@ -38,6 +39,7 @@ const state: State = {
   userId: 0,
   roles: [],
   tenant: '',
+  syncTableData: null,
```

Comment:
It is recommended not to put syncTableData in the store. I think it is not global.